apollo-router/src/plugins/coprocessor/supergraph.rs

```diff
@@ -53,6 +55,8 @@ pub(super) struct SupergraphResponseConf {
     pub(super) status_code: bool,
     /// Handles the response without waiting for the coprocessor to respond
     pub(super) detached: bool,
+    /// The url you'd like to offload processing to
+    pub(super) url: Option<String>,
```

Comment:
```suggestion
    #[schemars(with = "String")]
    pub(super) url: Option<url::Url>,
```

Reply:
To benefit from url check when deserializing

Reply:
oh nice